### PR TITLE
Hides network change on GIVeconomy for multisig

### PR DESCRIPTION
**Pull request by mateodaza**

### CODE REVIEW

**Review by MohammadPCh — APPROVED** [review]
Thanks @mateodaza

**Comment by MohammadPCh** [comment]
@mateodaza I don't merge it as our new workflow. the second review who checks the functionality can merge it 💜

**Comment by mateodaza** [comment]
> @mateodaza I don't merge it as our new workflow. the second review who checks the functionality can merge it 💜

Requesting @RamRamez review as well!

**Review by RamRamez — APPROVED** [review]

**Pull request merged**
